pom.xml

```diff
@@ -72,7 +72,7 @@
     <avro.version>1.8.2</avro.version>
     <bigquery.connector.hadoop2.version>hadoop2-1.0.0</bigquery.connector.hadoop2.version>
     <commons.codec.version>1.4</commons.codec.version>
-    <cdap.version>6.3.0-SNAPSHOT</cdap.version>
+    <cdap.version>6.2.3</cdap.version>
```

Comment:
@ardian4 do you have any comment about changing this from 6.3.0-SNAPSHOT to 6.2.3?
Does the PR needs to have POM changes?